### ISSUE TITLE
Display user info in dashboard

### ISSUE THREAD

**Issue by jyeatman** [yeatmanlab/roar-dashboard#96]
I can't tell which account I'm logged in with. We should display user info in an obvious place on the homescreen. This will be important in many applications where a teacher, for example, will want to be able to walk by and see in a glance if someone is logged in to the right account. I think displaying basic user info like username, first name, last name, grade, age, DOB, classroom and any other user group info would be good. I don't have a strong opinion on how or where other than that it should a) look nice and b) be static and easy to see at a glance

Tagging @dguimara for his thoughts. Let's start a conversation as this is not a pressing issue but an important enhancement

**Comment by dguimara** [comment]
@jyeatman yeah this is a great idea. We already have a spot in the sidebar for this - we currently have just a few details but we could easily expand this to include more information. 

I'm thinking we include:
- First & Last name OR username (when first/last aren't available)
- Grade
- Classroom (or school or district, depending on what is available)

Age/DOB feel less important and redundant if we have the above, and especially if we have First & Last name.

**Comment by Emily-ejag** [comment]
I believe this is done
